examples/blogs-videos/max-optimize-deploy/client-max-webcam.py

```diff
@@ -0,0 +1,90 @@
+import cv2
+import numpy as np
+import tritonclient.http as httpclient
+import tensorflow as tf
+import json
+import argparse
+import time
+
+width = 1280
+height = 720
+scale_factor = 0.2
+text_color = (0, 0, 255)
+
+### Image pre-processing ###
```

Comment:
Is this comment needed? The name of the method is already self explanatory

Reply:
Appreciate the feedback! We'll address them after the blog post goes live!